src/components/SelectedCardList/SelectedCard/SelectedCard.tsx

```diff
@@ -1,19 +1,25 @@
+import React, { useContext } from "react";
+import { UriContext } from "../../../common";
+import  "./SelectedCard.css";
 
 type Props = {
   uuid: number,
   cardName: string,
   secondName: string,
   cardType: string,
+  cardImage: string
   onDeleteItem: (uuid: number) => void
 }
 
 
 const SelectedCard: React.FC<Props> = (props: Props) => {
-
-  return <>
-    <p>{props.cardName} / {props.cardType} / {props.secondName}</p>
-    <button onClick={() => props.onDeleteItem(props.uuid)}>remove card</button>
-  </>
+  const { uuid, cardName, secondName,cardType, cardImage } = props;
```

Comment:
```suggestion
  const { uuid, cardName, secondName, cardImage } = props;
```
`cardType` 은 쓰고 있지 않는 것 같은데 지워도 되지 않을까요?

Reply:
제가 해당 게임에 대한 도메인 지식이 없긴 한데, 저 카드 타입이 캐릭터 속성을 나타내는 거라면, border 색상을 저걸 참조해서 그리면 좋지 않을까요? 검색 UI에서도 card type에 해당하는 필터링 칩을 추가해 볼 여지도 있을 것 같습니다

Reply:
아 네, 하지만 지금은 없으니까... webpack에서 이런 미사용 변수들이 있을 경우 에러를 띄우는데 에러가 잡히면 netlify에서 빌드가 안 됩니다 ㅠㅠ

Reply:
아 아래 넷틀리파이 배포 실패가 뜨고 있었군요 ㅠㅠ 우선은 지워두시고 오늘 작업분에 넣도록 하겠습니다